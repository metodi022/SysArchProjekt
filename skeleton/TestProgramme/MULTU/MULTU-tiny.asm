lui $1, 0x0000  // 32'd5
ori $1, 0x0005
lui $2, 0x0000  // 32'd10
ori $2, 0x000a
multu $1, $2    // multiply $1 and $2
mfhi $3         // 0x00000000
mflo $4         // 0x00000032

// 5*10 = 0x5 * 0xA = 0x32
// expected: 0x5fffffff
