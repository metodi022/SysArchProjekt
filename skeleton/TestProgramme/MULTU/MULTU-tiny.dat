0x3c010000
0x34210005
0x3c020000
0x3442000a
0x00220019
0x00001810
0x00002012
