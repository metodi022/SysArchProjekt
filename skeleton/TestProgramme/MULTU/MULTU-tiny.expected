@1 00000005
@2 0000000a
@3 00000000
@4 00000032
